presto-docs/src/main/sphinx/release/release-326.rst

```diff
@@ -0,0 +1,46 @@
+=========================
+Release 326 (25 Nov 2019)
+=========================
+
+General Changes
+---------------
+
+* Fix performance regression in queries involving ``JOIN``. (:issue:`2047`)
+* Fix accounting of semantic analysis time when queries are cancelled while queued. (:issue:`2055`)
+* Add :doc:`/connector/memsql`. (:issue:`1906`)
+* Improve performance of ``INSERT`` and ``CREATE TABLE ... AS`` queries containing redundant
+  ``ORDER BY`` clauses. (:issue:`2044`)
```

Comment:
Would be good to direct people towards `sorted_by` Hive table property

Reply:
Suggestions on wording? Not sure how to fit that in the context of this "engine-level" item.